service/project.go

```diff
@@ -595,3 +627,34 @@ func (uis *UIServer) setupGithubHook(projectRef *model.ProjectRef) (int, error)
 
 	return *hook.ID, nil
 }
+
+func (uis *UIServer) projectEvents(w http.ResponseWriter, r *http.Request) {
+	_ = MustHaveUser(r)
+	_ = MustHaveProjectContext(r)
```

Comment:
I believe this is redundant if you're getting users/project from gimlet

Reply:
I think you need to have it though. these just cause it to panic/not removed

Reply:
The code below will panic anyway if not set though

Reply:
removed